src/Draft/TranslationIndex.vue

```diff
@@ -251,5 +286,7 @@ onMounted(() => {
     title.value = model.getField('title', defaultTitle.value);
     saveDraft();
   });
+
+  sourceItemsLength = getSourceItemsLength(model.source);
```

Comment:
We probably do not have to wait for the widget to be mounted to calculate this. How about moving it out of here?